src/Hosting.ts

```diff
@@ -15,7 +15,8 @@ import { Drawer } from '@material/mwc-drawer';
 import { ActionDetail, List } from '@material/mwc-list';
 import { ListItem } from '@material/mwc-list/mwc-list-item';
 
-import { Mixin, newPendingStateEvent, UserInfoEvent } from './foundation.js';
+import { Mixin, newPendingStateEvent } from './foundation.js';
+import { UserInfoEvent } from './compas/foundation.js';
```

Comment:
Do we still want this in here? Or do we eventually move this out aswell (by extending Hosting in the compas package). Whatever we pick, this is still a step in the right direction!